### ISSUE TITLE
Version conflict: VASP 4.x/5.x documentation

### ISSUE THREAD

**Issue by jubich** [dftbplus/dftbplus#981]
The example in the manual uses VASP 4.x, but links to VASP 5.x.
The main difference is that in VASP 4.x the atom symbols are in line 1, while in VASP 5.x they are in line 6.

https://github.com/dftbplus/dftbplus/blob/5c110200a8b832a9e4f5148e1d6f9d644f1bada6/doc/dftb%2B/manual/dftbp.tex#L214-L227

**Comment by awvwgk** [comment]
DFTB+ can actually read both 4.x and 5.x formatted POSCAR files.

**Comment by jubich** [comment]
Yes, but I would say it is a bit misleading to give an example for VASP 4.x and link to VASP 5.x without mentioning that they are two different formats/versions or that DFTB+ can read both.

**Comment by jubich** [comment]
Ok, I will change it.
